File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.java
class Solution {
    public int maxCoins(int[] piles) {
        Arrays.sort(piles);
        int i = piles.length - 2;
        int j = (int)piles.length/3;
        int maxCoins = 0;
        
        while(i >= j){
            maxCoins += piles[i];
            i -= 2;
        }
        
        return maxCoins;
    }
}

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        sortedPiles = sorted(piles, reverse=True)
        sum = 0
        n = len(piles)
        m = 1
        while m < n - (n//3):
            sum += sortedPiles[m]
            m += 2
        return sum